dashboard/app/models/queued_account_purge.rb

```diff
@@ -31,6 +31,13 @@
 class QueuedAccountPurge < ApplicationRecord
   belongs_to :user
 
+  # Some errors are known to be intermittent, such as external services being temporarily
+  # unavailable. If an account purge was queued for one of these reasons, our system can
+  # automatically retry it on the next run without developer intervention.
+  # @return [Boolean] Whether this queued purge can be auto-retried.
```

Comment:
:+1: for the explanatory comment, but the `@return` line seems out of place

Reply:
:man_facepalming: Leftover from an earlier implementation using a method instead of a scope.  Fixing...